src/FFmpeg.NET/FFmpegArgumentBuilder.cs

```diff
@@ -94,12 +94,16 @@ private static string Convert(MediaFile inputFile, MediaFile outputFile, Convers
                 commandBuilder.AppendFormat(" -t {0} ", conversionOptions.MaxVideoDuration);
 
             // Video bit rate
+            if (conversionOptions.VideoBitRate != null)
```

Comment:
VideoFps Option has been removed but VideoBitRate is duplicated.
This does not seem right to me